traffic_portal/app/src/common/api/JobService.js

```diff
@@ -31,7 +31,7 @@ var JobService = function($http, ENV) {
 	};
 
 	this.createJob = function(job) {
-		return $http.post(ENV.api['root'] + 'user/current/jobs', job).then(
+		return $http.post(ENV.api['root'] + 'jobs', job).then(
 			function (result) {
 				console.info('new job created: ', result);
```

Comment:
want to remove this console line?

Reply:
I guess. Idk if having that is really a problem, but it's gone.